src/main/webapp/drugs/singleDrug.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Now that you have a couple of examples of this template for particular drugs, it would be worth trying to convert this into a React component where the drug-specific parts are properties of the component.